docs/reference/release-notes/7.9.asciidoc

```diff
@@ -37,3 +37,281 @@ Dangling indices::
   default, and will be removed in {es} 8.0. See the
   <<deprecate_auto_import_dangling_indices,migration notes>>.
   {es-pull}58176[#58176] {es-pull}58898[#58898] (issue: {es-issue}48366[#48366])
+
```

Comment:
I think a "coming::[7.9.0]" should be added at the top of this page too

Reply:
Added this.